features/step_definitions/navigation_steps.rb

```diff
@@ -406,6 +406,14 @@
     )
 end
 
+Then(/^I should see no system apart from the proxy$/) do
+  if $proxy.nil?
```

Comment:
use guard clause

Reply:
http://www.thechrisoshow.com/2009/02/16/using-guard-clauses-in-your-ruby-code/

Reply:
Thanks. Fixed this one with a ternary, looks more adapted to me.

Reply:
honestly, for that one, a guard clause is less readable and does not bring anything